template/manifests/base/virtualservice.yaml

```diff
@@ -0,0 +1,22 @@
+### Uncomment this if you're exposing a public API. Remove this file otherwise.
+
+# apiVersion: networking.istio.io/v1alpha3
+# kind: VirtualService
+# metadata:
+#   name: {{Name}}-vs
```

Comment:
The app and project labels should be added here too. Along with all of the other resources in manifests/base/.